docs/vars.md

```diff
@@ -67,6 +67,29 @@ following default cluster parameters:
   (default is k8s_external.local)
 * *enable_coredns_k8s_endpoint_pod_names* - If enabled, it configures endpoint_pod_names option for kubernetes plugin.
   on the CoreDNS service.
+* *coredns_external_zones* - Array of optional external zones to coredns forward queries to. It's  injected into
```

Comment:
This is a great patch, but this doc file is for short descriptions only. Can you move this text to docs/dns-stack.md?

Reply:
Applied your suggestion. Thanks for reviewing.